src/View/Helper/ColorHelper.php

```diff
@@ -28,7 +28,7 @@ public function hexToRgb(string $hexCode)
         if ($hexCode[0] == '#') {
             $hexCode = substr($hexCode, 1);
         }
-        if (strlen($hexCode) < 4 && strlen($hexCode) >2) { // WTF phpstan for php7.4???
+        if (!$hexCode[4]) { // WTF phpstan for php7.4???
```

Comment:
Ok, this has to be the stupidest fail for PHPstan that I've seen. How do you know the specific output of strlen??